### DISCUSSION TITLE
安卓版可否增加移除动画功能？

### DISCUSSION THREAD

**Discussion by llllllllqq** [lollipopkit/flutter_server_box#622] [Ideas]
因为hyperos的键盘打开/收起的动画非常慢&弹来弹去很别扭，希望增加个app内置的关闭动画的开关

**Comment by GT-610** [comment]
这个是系统的因素，不是程序的问题，程序内部所有动画都是通过Flutter组件绘制的，此部分我们已确保动画设计符合逻辑。
